auto/cargo

```diff
@@ -57,10 +57,22 @@ if [ $ngx_wasm_rc = 0 ]; then
                 $ngx_feature_libs"
 
         elif [ "$NGX_WASM_RUNTIME_NO_RPATH" != 1 ]; then
-            ngx_feature_libs="\
-                -Wl,-rpath=$ngx_wasm_cargo_target_dir \
-                -Wl,-rpath=$NGX_PREFIX/lib \
-                $ngx_feature_libs"
+            case "$NGX_PLATFORM" in
+                Darwin:*)
+                    ngx_feature_libs="\
+                        -Wl,-rpath,$ngx_wasm_cargo_target_dir \
+                        -Wl,-rpath,$NGX_PREFIX/lib \
+                        $ngx_feature_libs"
+                ;;
+
+                *)
+                    ngx_feature_libs="\
+                        -Wl,-rpath=$ngx_wasm_cargo_target_dir \
+                        -Wl,-rpath=$NGX_PREFIX/lib \
+                        $ngx_feature_libs"
+                ;;
+
+            esac
```

Comment:
What is the purpose of this switch? I can't spot a difference between the 2 branches.

Reply:
Clang expects `-rpath` to be provided as `-rpath,/target_dir` while gcc expects it to be provided as `-rpath=/target_dir`.